pkgs/by-name/no/noto-fonts-cjk-serif/package.nix

```diff
@@ -3,23 +3,44 @@
 , fetchFromGitHub
 , nixosTests
 , gitUpdater
+
+, static ? false, # whether to build the static version of the font
 }:
 
 stdenvNoCC.mkDerivation rec {
   pname = "noto-fonts-cjk-serif";
   version = "2.003";
 
-  src = fetchFromGitHub {
-    owner = "notofonts";
-    repo = "noto-cjk";
-    rev = "Serif${version}";
-    hash = "sha256-E+Ic7XhomI6cUa+q77gQvMlaLvy+vgTq4NJ58/nPZtk=";
-    sparseCheckout = [ "Serif/Variable/OTC" ];
-  };
+  src =
+    let
+      sources = {
+        static = fetchFromGitHub {
+          owner = "notofonts";
+          repo = "noto-cjk";
+          rev = "Serif${version}";
+          hash = "sha256-mfbBSdJrUCZiUUmsmndtEW6H3z6KfBn+dEftBySf2j4=";
+          sparseCheckout = [ "Serif/OTC" ];
+        };
+        variable = fetchFromGitHub {
+          owner = "notofonts";
+          repo = "noto-cjk";
+          rev = "Serif${version}";
+          hash = "sha256-E+Ic7XhomI6cUa+q77gQvMlaLvy+vgTq4NJ58/nPZtk=";
+          sparseCheckout = [ "Serif/Variable/OTC" ];
+        };
+      };
+    in
+    if static then sources.static else sources.variable;
 
-  installPhase = ''
-    install -m444 -Dt $out/share/fonts/opentype/noto-cjk Serif/Variable/OTC/*.otf.ttc
-  '';
+  installPhase =
+    if static then
+      ''
+        install -m444 -Dt $out/share/fonts/opentype/noto-cjk Serif/OTC/*.ttc
+      ''
+    else
+      ''
+        install -m444 -Dt $out/share/fonts/opentype/noto-cjk Serif/Variable/OTC/*.otf.ttc
+      '';
```

Comment:
Since a good part of the command is shared, you may also prefer to inline the expression:

```nix
install -m444 -Dt $out/share/fonts/opentype/noto-cjk ${if static then "Serif/OTC/*.ttc" else "Serif/Variable/OTC/*.otf.ttc"}
```

(or use a `let` for the files to copy to make it clearer)